packages/common/test/TransactionUtils.js

```diff
@@ -27,5 +28,23 @@ describe("TransactionUtils.js", function () {
       }
     });
     // TODO: Figure out how to test situations where the `transaction.send()` fails but `.call()` does not
+    it("waitForMine = false doesn't hang if transaction hash errors", async function () {
+      const erc20Contract = await ERC20.new("1").send({ from: accounts[0] });
```

Comment:
I've never seen this syntax before to initiate the ERC20 with 1 token for the caller